Documentation/source/Updating to 4.0.md

```diff
@@ -0,0 +1,2 @@
+Updating to 4.0
+=============================
```

Comment:
![maxresdefault](https://user-images.githubusercontent.com/13149498/41861315-2efd50ca-78a1-11e8-9f42-aafa98403a30.jpg)

Reply:
I decided to tackle this in a different PR, that's way I only created the reference here.